lib/xsdParser.coffee

```diff
@@ -147,6 +147,9 @@ module.exports =
     else if node.union
       type.xsdChildrenMode = 'union'
       type.leftLabel = node.union[0].$.memberTypes
+    else if node.list
```

Comment:
Can you remove the comment a few lines above that says `TODO: Support list children`?

Reply:
Fixed and uploaded to branch.